src/libraries/Microsoft.Extensions.Hosting.Abstractions/src/BackgroundService.cs

```diff
@@ -78,6 +91,10 @@ public virtual async Task StopAsync(CancellationToken cancellationToken)
 
         }
 
+        /// <summary>
+        /// Tells the service task to stop but does not wait for it to stop.
+        /// </summary>
+
```

Comment:
remove empty blank line (here and in two more instances above)

Reply:
> @carlossanlop will changes made here flow successfully to docs pages or does another PR need to mirror changes here into [dotnet docs API](https://github.com/dotnet/dotnet-api-docs) repo?

@maryamariyan If a comment section in the dotnet-api-docs xml currently contains the boiler plate string `To be added.`, then new triple slash comments will become portable. But if in the dotnet-api-docs xml the text was already filled out with something other that that boiler plate string, then any changes added to triple slash won't flow, and you'll have to edit the dotnet-api-docs xml manually.